shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Examples/Consensus.hs

```diff
@@ -63,7 +65,7 @@ type KeyPairWits era = [KeyPair 'Witness (Cardano.Ledger.Era.Crypto era)]
 -------------------------------------------------------------------------------}
 
 data ShelleyResultExamples era = ShelleyResultExamples
-  { srePParams :: Shelley.Spec.Ledger.PParams.PParams era,
+  { srePParams :: Cardano.Ledger.Core.PParams era,
```

Comment:
@Jasagredo Are our downstream use-cases going to easily cope with this change from data type to type family?

Reply:
This is fine, it it was possible to generate the PParams for the other eras in this flavor, then we are all good.